source/index.html.md

```diff
@@ -174,12 +173,20 @@ We are interested in the message parameter. We simply check if it contains the w
 
 ## Module Organization
 
-When making a module, it is important to know how ct.js loads your files so you can organize them efficiently. When your module is loaded, the root folder and any subfolders are scanned for files of the correct type (for example, `.js` files) and are loaded as one big file. 
+> ES6 style import syntax (preferred)
 
-Those with previous programming experience with Javascript may be familiar with NodeJS's `require` system, where in order to get access to code in another file, you have to call a function and provide the file path. In ct.js, all of your module files are compiled into one big file and loaded at once. This means that if you define a global variable in one file, you can use it immediately in any other file in your module.
+```js
+// This imports a function from WhereAmI's index file, you could specify additional files i.e. WhereAmI/otherfile.js
+import playerLocation from 'WhereAmi';
```

Comment:
It might be worth noting that you can also import from the current module using local paths (i.e. `./folder/file`)